lib/application/widgets/theme/retry_container.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:layoutr/common_layout.dart';
+import 'package:memo/application/constants/exception_strings.dart';
+import 'package:memo/application/constants/strings.dart' as strings;
+import 'package:memo/application/widgets/theme/custom_button.dart';
+import 'package:memo/core/faults/exceptions/base_exception.dart';
+
+/// A container that wraps a localized [exception] with a button that, when pressed, calls [onRetry].
+///
+/// This widget is mostly used in scenarios where a page couldn't load its essential data, meaning that there is no
+/// alternative other than retry loading its contents.
+class ExceptionRetryContainer extends StatelessWidget {
+  const ExceptionRetryContainer({required this.onRetry, required this.exception});
+
+  final VoidCallback onRetry;
+  final BaseException exception;
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+
+    return Column(
```

Comment:
Will this fill the whole screen?

Reply:
<details>
<summary>Screenshot</summary>

![Uploading CleanShot 2021-11-10 at 15.28.44@2x.png…]()

</details>

Reply:
Shouldn't it be centered and fill all the available height?

Reply:
I'll update the component parent, because it's centered but the Column is not expanded in the parent